docs/user-guide/install-zowe-server-install-wizard.md

```diff
@@ -0,0 +1,226 @@
+
+## Prerequisites of the Wizard
+
+- Microsoft Windows, Apple Mac OS, or a Linux with an X11 or Wayland server display which can install programs from .rpm or .deb formats 
+- An FTP or FTPS connection to z/OS for Zowe installation
+- An account on z/OS that has access to z/OS UNIX for Zowe installation 
+- A security administrator to configure required permissions in z/OS and z/OSMF
+- A security administrator to generate certificates for Zowe
+- A network administrator to open ports used by Zowe 
+
+## Downloading the Wizard
+
+To download the latest version of the Wizard, visit [Zowe.org](https://www.zowe.org/download.html).
+Ensure that you download the appropriate file extension type according to your operating system:
+
+Operating System | File Extension Type
+---|---
+Microsoft Windows | .exe
+Apple Mac OS | .dmg
+Linux (debian-based) | .deb
+Linux (RedHat or SuSE-based) | .rpm
+
+
+## Installing Zowe server-side components
+
+1. Select **New Zowe Installation**.
+2. In the Connection window, provide details as described in the following table:
+
+### Setup Wizard connection to z/OS
+
+Field name| Description                
+---|---
+Host      |Value for the target z/OS system for Zowe Installation. For example, `mainframe.yourcompany.com`
+FTP Port  |The FTP Port number for internal use. The default port is 21. If not specified, the wizard uses the default port.
+User Name |Your z/OS username.
+Password  |Your z/OS password.
+
+3. Select Use FTP over TLS (Highly Recommended). Provide details as described in the following table:
+        
+Field name | Description
+---|---
+Min TLS    |Select the minimum TLS version to accept the certificate from the server.
+Max TLS    |Select the maximum TLS version to accept the certificate from the server.
+4. Select **Accept all certificates**.
+5. Click **VALIDATE CREDENTIALS**.
+6. Click **Continue**.
+7. In the **Before you start** window, review the instructions. 
+8. In the **Job statement** field, customize the job statement if needed and click **SAVE AND VALIDATE**.
+9. In the **Planning window**, provide details for z/OS Unix locations, identifiers, and z/OSMF details. 
+
+### Setting z/OSMF Attributes (optional)
+Provide details for z/OSMF. 
+
+1. After specifying all the mandatory details , click **VALIDATE LOCATIONS**.
+
+    You will receive a confirmation message for Java and Node.js locations.
+
+2. Click **Continue to Installation Options**.
+
+## Choosing the Server Installation Type
+
+1. In the **Installation Type** window, select one of the following three installation types in the Wizard:
+
+    <details>
+    <summary>Download Zowe convenience build PAX from internet</summary>
+
+    Download the latest Zowe convenience build in .PAX format from [zowe.org](https://zowe.org) using the wizard without visiting the website.  <br />       1. Click **License Agreement**. <br />2. On the End User License Agreement for Zowe page, click **AGREE**.
+    </details>
```

Comment:
Then --> Continue to "Unpax"
Then --> "Begin Download"

Then wait for statuses to finish. After they finish, move onto Step 2